.gitignore

```diff
@@ -22,3 +22,7 @@ src
 
 parser.out
 parsetab.py
+
```

Comment:
good for now. Ideally we can have a flag to suppress Genesis2 print statements so these files don't get created 🙂